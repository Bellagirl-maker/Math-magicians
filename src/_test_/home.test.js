import { screen, render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from '../components/home';
import '@testing-library/jest-dom/extend-expect';

// import renderer from 'react-test-renderer';

describe('Home component', () => {
  it('renders the homepage', () => {
    render(<Home />);
    expect(screen.getByText(/Welcome/)).toBeInTheDocument();
    expect(screen.getByText(/Lorem Ipsum/)).toBeInTheDocument();
  });
});

describe('Home snapshot', () => {
  test('Home snapshot should match dom', () => {
    const tree = renderer.create(<Home />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
