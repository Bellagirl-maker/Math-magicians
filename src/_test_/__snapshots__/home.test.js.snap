// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot Home snapshot should match dom 1`] = `[Function]`;
